ember-flight-icons/tests/integration/components/flight-icon-test.js

```diff
@@ -60,14 +52,12 @@ module('Integration | Component | flight-icon', function (hooks) {
       fill: 'rgb(186, 34, 38)',
     });
   });
-  // the component should have `color` set to `currentColor` (black) by default
   test('the fill should be set to black by default', async function (assert) {
```

Comment:
```suggestion
  test('the fill color should be set to black by default', async function (assert) {
```

suggestion to be consistent with test name below